components/shared/Header/Header.tsx

```diff
@@ -4,35 +4,46 @@ import { useNavOptions } from "../../hooks/useNavOptions";
 import { SignupButton } from "../../Homepage/SignupButton";
 import * as styles from "./Header.css";
 import { useSessionContext } from "@supabase/auth-helpers-react";
-import { Router } from "next/router";
 import { useRouter } from "next/router";
+import { Button, useDisclosure } from "@chakra-ui/react";
+import { EmailAuthModal } from "components/EmailAuthModal";
 
 export const Header = (): ReactElement => {
   const { howItWorks } = useNavOptions();
   const router = useRouter();
+  const { isOpen, onOpen, onClose } = useDisclosure();
 
-  const { session, supabaseClient } = useSessionContext();
+  const { session, supabaseClient, isLoading } = useSessionContext();
   const signOut = async () => {
     await supabaseClient.auth.signOut();
   };
+
   return (
     <header id="header" className={styles.header}>
       <Link href={"/"}>
         <span className={styles.titleText}>Everyone Plays the Same Song</span>
       </Link>
       <div className={styles.navButtons}>
-        <Link href={howItWorks}>
-          <button>The rules</button>
-        </Link>
-        <SignupButton />
-        {process.env.NODE_ENV === "development" && (
-          <button
-            onClick={session ? () => signOut() : () => router.push("/la-cueva")}
-          >
-            {session ? "Sign Out" : "Sign in"}
-          </button>
+        <Button colorScheme="blue" variant="ghost" onClick={() => router.push("/how-it-works")}>
+          Rules
+        </Button>
+
+        {isLoading ? null : session ? (
```

Comment:
Can we keep this behind a dev flag for the time being? Idk if the profile itself is ready for showbiz? `process.env.NODE_ENV === "development"`